evalml/tests/model_understanding_tests/test_partial_dependence.py

```diff
@@ -394,25 +394,25 @@ def round_dict_keys(dictionary, places=6):
             dictionary_rounded[round(key, places)] = dictionary[key]
         return dictionary_rounded
 
-    X, y = load_fraud(1000)
+    X, y = load_fraud(100)
     X = X.drop(columns=["datetime", "expiration_date", "country", "region", "provider"])
     pipeline = logistic_regression_binary_pipeline_class({})
     pipeline.fit(X, y)
     num_cat_features = len(set(X["currency"]))
-    assert num_cat_features == 164
+    assert num_cat_features == 73
 
     part_dep_ans = {
-        0.1432616813857269: 154,
-        0.1502346349971562: 1,
-        0.14487916687594762: 1,
-        0.1573183451314127: 1,
-        0.11695462432136654: 1,
-        0.07950579532536253: 1,
-        0.006794444792966759: 1,
-        0.17745270478939879: 1,
-        0.1666874487986626: 1,
-        0.13357573073236878: 1,
-        0.06778096366056789: 1,
+        0.05824028901694482: 63,
```

Comment:
Since the data we use is different, the expected partial dependence is now different too

Reply:
Honestly, I'm still not sure I understand the usefulness of checking against `part_dep_ans`... how are we getting these numbers anyhow 🤔